en/docs/setup/iam-feature-maintenance-policy.md

```diff
@@ -0,0 +1,99 @@
+# IAM Feature Maintenance Policy
+
+## Feature Deprecation
+
+Features/APIs in the server can be deprecated for several reasons such as
+
+*   New specification versions have been released
+*   Standard alternatives available OOTB
+*   Security risks involved
+
+### Feature Deprecation Life-Cycle
+
+While deprecated features can still be available and support for existing users, they will not be further enhanced, nor recommended for use. Removal of deprecated capabilities will occur over a deprecation window, in which the feature will undergo several phases in the deprecation life-cycle. 
+
+![feature-deprecation-life-cycle](../assets/img/setup/feature-deprecation-life-cycle.png)
+
+The following terms refer to the different phases of the feature deprecation life cycle.
+
+1. **Planned**
+
+    Any feature in the Planned phase, will be deprecated and removed in future releases. However, the feature is available for use in the current release.
+
+2. **Deprecated**
+
+    In the deprecated phase, we no longer recommend using the feature. While deprecated classes and methods are still available, they are tagged deprecated and are disabled by default in the product, because they will be removed in future releases. This is done to facilitate the phasing-out for any existing user that still uses the deprecated features for their use-cases. It is not recommended to use them for implementing any new use cases/extensions. We encourage you to start upgrading any existing code written using the deprecated features during this phase.
+
+3. **Retired**
+
+    At this phase, we decide not to incorporate the feature with new releases. For versions released, until the respective version goes end of license, feature maintenance and security updates will be provided for subscriptions
```

Comment:
In the deprecation matrix, these are not the labels that have been used. In the matrix we have used a label called "FUTURE" and we have not used labels like "Planned".

Let's make this consistent, otherwise it will be confusing to the user.

Reply:
Shall we change the term "FUTURE" to "PLANNED" 
WDYT @malithie ?

Reply:
We can add a special note in the deprecation matrix saying ;
- FUTURE tag is used to indicate deprecation/retired timelines which are not yet finalized.
- FUTURE on 'deprecated in' column indicate that the feature is in 'Planned'  phase however deprecated timeline is not yet finalized.

WDYT @sherenewso2 @malithie

Reply:
So we are referring to a release milestone here. I think it would be good to define the tag before, and what we mention in the column.
Like the matrix includes the product release versions that each feature will be deprecated or retired from. If "FUTURE" it means ...

Reply:
Also, link the support matrix to relate to the EOL dates of a product

Reply:
Fixed 29aa1c7